packages/radix-vue/src/Stepper/StepperSeparator.vue

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+import { injectStepperRootContext } from './StepperRoot.vue'
+import { Separator, type SeparatorProps } from '@/Separator'
+import { useForwardExpose } from '@/shared'
+</script>
+
+<script setup lang="ts">
+export interface StepperSeparatorProps extends SeparatorProps { }
+
+const props = withDefaults(defineProps<StepperSeparatorProps>(), {})
+
+const rootContext = injectStepperRootContext()
+
+useForwardExpose()
+</script>
+
+<template>
+  <Separator
+    v-bind="props"
+    decorative
+    :orientation="rootContext.orientation.value"
+  >
```

Comment:
Since the `Separator` is a child of `Item`, maybe we can inject the `itemState` from `Item`? This way we can have the `data-state` auto populated as well. Thought? 😁